src/snapshots/can_import_nested_modules.md

```diff
@@ -420,7 +420,10 @@ LowerIdent(26:1-26:13),OpAssign(26:14-26:15),OpBar(26:16-26:17),LowerIdent(26:17
 module []
 
 import json.Parser
-import http.Clientutils[padLeft]
+
+import http.Client
+utils
+[padLeft]
```

Comment:
This formatting looks like a regression

Reply:
This is caused by [these](https://github.com/JRI98/roc/blob/fe19ff4a44938df9f3efcb84ee6eacd0edfe8faa/src/snapshots/can_import_nested_modules.md?plain=1#L335-L337) malformed statements and is a separate issue where `exposing` is not being kept in the formatted code. Will be fixed when that is actually kept in the output.
For reference, this is the original line:
```
import utils.String.Format exposing [padLeft]
```